libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/InstanceState.java

```diff
@@ -0,0 +1,53 @@
+package com.mapbox.services.android.navigation.ui.v5;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.design.widget.BottomSheetBehavior;
+
+public class InstanceState implements Parcelable {
```

Comment:
I think we can be more specific on the naming here, something like `NavigationViewInstanceState`?

Also, are we able to remove the `public` access?  This is only going to be used in `NavigationView`.